webpack.config.js

```diff
@@ -447,6 +447,19 @@ module.exports = async (env, argv) => {
           exclude: [path.resolve(__dirname, "node_modules")],
           loader: "babel-loader"
         },
+        {
+          test: [path.resolve(
+            __dirname,
+            "node_modules",
+            "three",
+            "examples",
+            "jsm",
+            "exporters",
+            "GLTFExporter.js"
+	  )],
+          loader: "babel-loader",
+          options: legacyBabelConfig
```

Comment:
Still slightly confused why this needed the "legacy" loader config... but I guess its fine. We definitely need to do a pass on cleaning up and updating our bundling.